Parsers/ASimAuditEvent/Parsers/vimAuditEventCiscoISE.yaml

```diff
@@ -0,0 +1,360 @@
+Parser:
+  Title: Audit Event ASIM filtering parser for Cisco ISE
+  Version: '0.1'
+  LastUpdated: July 13, 2023
+Product:
+  Name: Cisco ISE
+Normalization:
+  Schema: AuditEvent
+  Version: '0.1.0'
+References:
+- Title: ASIM Audit Event Schema
+  Link: https://aka.ms/ASimAuditEventDoc
+- Title: ASIM
+  Link: https:/aka.ms/AboutASIM
+- Title: Cisco ISE Security Events 
+  Link: https://www.cisco.com/c/en/us/td/docs/security/ise/3-2/admin_guide/b_ise_admin_3_2/b_ISE_admin_32_maintain_monitor.html#ID58
+Description: |
+  This ASIM parser supports normalizing administrative activity in the Cisco ISE events to the ASIM Audit Event schema.
+ParserName: ASimAuditEventCiscoISE
```

Comment:
@spsocprime can you please change this to vimAuditEventCiscoISE?

Reply:
changed